2022-06-25 14:11:04,060 [ERROR] from com.jpro.internal.server.ViewSerializerActor in JavaFX Application Thread - JavaFXStreamer - got Exception: 
java.lang.AssertionError: assertion failed: Couldn't find the instance for: AppByID(808093390,None)
	at scala.Predef$.assert(Predef.scala:223)
	at com.jpro.internal.config.ApplicationStarter$.getApplication(ApplicationStarter.scala:56)
	at com.jpro.internal.config.ApplicationStarter$.$anonfun$getApplication$2(ApplicationStarter.scala:54)
	at scala.Option.map(Option.scala:230)
	at com.jpro.internal.config.ApplicationStarter$.appor$lzycompute$1(ApplicationStarter.scala:54)
	at com.jpro.internal.config.ApplicationStarter$.appor$1(ApplicationStarter.scala:54)
	at com.jpro.internal.config.ApplicationStarter$.$anonfun$getApplication$3(ApplicationStarter.scala:55)
	at scala.Option.orElse(Option.scala:447)
	at com.jpro.internal.config.ApplicationStarter$.getApplication(ApplicationStarter.scala:55)
	at com.jpro.internal.communicator.ViewImpl.<init>(ViewImpl.scala:30)
	at com.jpro.internal.server.ViewByActor.<init>(ViewSerializerActor.scala:116)
	at com.jpro.internal.server.ViewSerializerActor$$anonfun$receive$2$$anonfun$applyOrElse$1.$anonfun$applyOrElse$2(ViewSerializerActor.scala:70)
	at simplefx.cores.standard.parts.CoreTimeEvent$$anon$3.run(CoreTimeEvent.scala:62)
	at com.sun.javafx.application.PlatformImpl.lambda$runLater$10(PlatformImpl.java:457)
	at java.base/java.security.AccessController.doPrivileged(Native Method)
	at com.sun.javafx.application.PlatformImpl.lambda$runLater$11(PlatformImpl.java:456)
	at com.sun.glass.ui.monocle.RunnableProcessor.runLoop(RunnableProcessor.java:92)
	at com.sun.glass.ui.monocle.RunnableProcessor.run(RunnableProcessor.java:51)
	at java.base/java.lang.Thread.run(Thread.java:834)
2022-06-25 14:11:04,107 [ERROR] from com.jpro.internal.server.ViewSerializerActor in JavaFX Application Thread - JavaFXStreamer - got Exception: 
java.lang.AssertionError: assertion failed: Couldn't find the instance for: AppByID(140726440,None)
	at scala.Predef$.assert(Predef.scala:223)
	at com.jpro.internal.config.ApplicationStarter$.getApplication(ApplicationStarter.scala:56)
	at com.jpro.internal.config.ApplicationStarter$.$anonfun$getApplication$2(ApplicationStarter.scala:54)
	at scala.Option.map(Option.scala:230)
	at com.jpro.internal.config.ApplicationStarter$.appor$lzycompute$1(ApplicationStarter.scala:54)
	at com.jpro.internal.config.ApplicationStarter$.appor$1(ApplicationStarter.scala:54)
	at com.jpro.internal.config.ApplicationStarter$.$anonfun$getApplication$3(ApplicationStarter.scala:55)
	at scala.Option.orElse(Option.scala:447)
	at com.jpro.internal.config.ApplicationStarter$.getApplication(ApplicationStarter.scala:55)
	at com.jpro.internal.communicator.ViewImpl.<init>(ViewImpl.scala:30)
	at com.jpro.internal.server.ViewByActor.<init>(ViewSerializerActor.scala:116)
	at com.jpro.internal.server.ViewSerializerActor$$anonfun$receive$2$$anonfun$applyOrElse$1.$anonfun$applyOrElse$2(ViewSerializerActor.scala:70)
	at simplefx.cores.standard.parts.CoreTimeEvent$$anon$3.run(CoreTimeEvent.scala:62)
	at com.sun.javafx.application.PlatformImpl.lambda$runLater$10(PlatformImpl.java:457)
	at java.base/java.security.AccessController.doPrivileged(Native Method)
	at com.sun.javafx.application.PlatformImpl.lambda$runLater$11(PlatformImpl.java:456)
	at com.sun.glass.ui.monocle.RunnableProcessor.runLoop(RunnableProcessor.java:92)
	at com.sun.glass.ui.monocle.RunnableProcessor.run(RunnableProcessor.java:51)
	at java.base/java.lang.Thread.run(Thread.java:834)
2022-06-25 14:11:04,217 [ERROR] from com.jpro.internal.server.ViewSerializerActor in JavaFX Application Thread - JavaFXStreamer - got Exception: 
java.lang.AssertionError: assertion failed: Couldn't find the instance for: AppByID(1452336998,None)
	at scala.Predef$.assert(Predef.scala:223)
	at com.jpro.internal.config.ApplicationStarter$.getApplication(ApplicationStarter.scala:56)
	at com.jpro.internal.config.ApplicationStarter$.$anonfun$getApplication$2(ApplicationStarter.scala:54)
	at scala.Option.map(Option.scala:230)
	at com.jpro.internal.config.ApplicationStarter$.appor$lzycompute$1(ApplicationStarter.scala:54)
	at com.jpro.internal.config.ApplicationStarter$.appor$1(ApplicationStarter.scala:54)
	at com.jpro.internal.config.ApplicationStarter$.$anonfun$getApplication$3(ApplicationStarter.scala:55)
	at scala.Option.orElse(Option.scala:447)
	at com.jpro.internal.config.ApplicationStarter$.getApplication(ApplicationStarter.scala:55)
	at com.jpro.internal.communicator.ViewImpl.<init>(ViewImpl.scala:30)
	at com.jpro.internal.server.ViewByActor.<init>(ViewSerializerActor.scala:116)
	at com.jpro.internal.server.ViewSerializerActor$$anonfun$receive$2$$anonfun$applyOrElse$1.$anonfun$applyOrElse$2(ViewSerializerActor.scala:70)
	at simplefx.cores.standard.parts.CoreTimeEvent$$anon$3.run(CoreTimeEvent.scala:62)
	at com.sun.javafx.application.PlatformImpl.lambda$runLater$10(PlatformImpl.java:457)
	at java.base/java.security.AccessController.doPrivileged(Native Method)
	at com.sun.javafx.application.PlatformImpl.lambda$runLater$11(PlatformImpl.java:456)
	at com.sun.glass.ui.monocle.RunnableProcessor.runLoop(RunnableProcessor.java:92)
	at com.sun.glass.ui.monocle.RunnableProcessor.run(RunnableProcessor.java:51)
	at java.base/java.lang.Thread.run(Thread.java:834)
2022-06-25 14:11:07,658 [ERROR] from com.jpro.internal.server.ViewSerializerActor in JavaFX Application Thread - JavaFXStreamer - got Exception: 
java.lang.AssertionError: assertion failed: Couldn't find the instance for: AppByID(8511801,None)
	at scala.Predef$.assert(Predef.scala:223)
	at com.jpro.internal.config.ApplicationStarter$.getApplication(ApplicationStarter.scala:56)
	at com.jpro.internal.config.ApplicationStarter$.$anonfun$getApplication$2(ApplicationStarter.scala:54)
	at scala.Option.map(Option.scala:230)
	at com.jpro.internal.config.ApplicationStarter$.appor$lzycompute$1(ApplicationStarter.scala:54)
	at com.jpro.internal.config.ApplicationStarter$.appor$1(ApplicationStarter.scala:54)
	at com.jpro.internal.config.ApplicationStarter$.$anonfun$getApplication$3(ApplicationStarter.scala:55)
	at scala.Option.orElse(Option.scala:447)
	at com.jpro.internal.config.ApplicationStarter$.getApplication(ApplicationStarter.scala:55)
	at com.jpro.internal.communicator.ViewImpl.<init>(ViewImpl.scala:30)
	at com.jpro.internal.server.ViewByActor.<init>(ViewSerializerActor.scala:116)
	at com.jpro.internal.server.ViewSerializerActor$$anonfun$receive$2$$anonfun$applyOrElse$1.$anonfun$applyOrElse$2(ViewSerializerActor.scala:70)
	at simplefx.cores.standard.parts.CoreTimeEvent$$anon$3.run(CoreTimeEvent.scala:62)
	at com.sun.javafx.application.PlatformImpl.lambda$runLater$10(PlatformImpl.java:457)
	at java.base/java.security.AccessController.doPrivileged(Native Method)
	at com.sun.javafx.application.PlatformImpl.lambda$runLater$11(PlatformImpl.java:456)
	at com.sun.glass.ui.monocle.RunnableProcessor.runLoop(RunnableProcessor.java:92)
	at com.sun.glass.ui.monocle.RunnableProcessor.run(RunnableProcessor.java:51)
	at java.base/java.lang.Thread.run(Thread.java:834)
2022-06-25 14:11:07,699 [ERROR] from com.jpro.internal.server.ViewSerializerActor in JavaFX Application Thread - JavaFXStreamer - got Exception: 
java.lang.AssertionError: assertion failed: Couldn't find the instance for: AppByID(140726440,None)
	at scala.Predef$.assert(Predef.scala:223)
	at com.jpro.internal.config.ApplicationStarter$.getApplication(ApplicationStarter.scala:56)
	at com.jpro.internal.config.ApplicationStarter$.$anonfun$getApplication$2(ApplicationStarter.scala:54)
	at scala.Option.map(Option.scala:230)
	at com.jpro.internal.config.ApplicationStarter$.appor$lzycompute$1(ApplicationStarter.scala:54)
	at com.jpro.internal.config.ApplicationStarter$.appor$1(ApplicationStarter.scala:54)
	at com.jpro.internal.config.ApplicationStarter$.$anonfun$getApplication$3(ApplicationStarter.scala:55)
	at scala.Option.orElse(Option.scala:447)
	at com.jpro.internal.config.ApplicationStarter$.getApplication(ApplicationStarter.scala:55)
	at com.jpro.internal.communicator.ViewImpl.<init>(ViewImpl.scala:30)
	at com.jpro.internal.server.ViewByActor.<init>(ViewSerializerActor.scala:116)
	at com.jpro.internal.server.ViewSerializerActor$$anonfun$receive$2$$anonfun$applyOrElse$1.$anonfun$applyOrElse$2(ViewSerializerActor.scala:70)
	at simplefx.cores.standard.parts.CoreTimeEvent$$anon$3.run(CoreTimeEvent.scala:62)
	at com.sun.javafx.application.PlatformImpl.lambda$runLater$10(PlatformImpl.java:457)
	at java.base/java.security.AccessController.doPrivileged(Native Method)
	at com.sun.javafx.application.PlatformImpl.lambda$runLater$11(PlatformImpl.java:456)
	at com.sun.glass.ui.monocle.RunnableProcessor.runLoop(RunnableProcessor.java:92)
	at com.sun.glass.ui.monocle.RunnableProcessor.run(RunnableProcessor.java:51)
	at java.base/java.lang.Thread.run(Thread.java:834)
2022-06-25 14:11:07,722 [ERROR] from com.jpro.internal.server.ViewSerializerActor in JavaFX Application Thread - JavaFXStreamer - got Exception: 
java.lang.AssertionError: assertion failed: Couldn't find the instance for: AppByID(1452336998,None)
	at scala.Predef$.assert(Predef.scala:223)
	at com.jpro.internal.config.ApplicationStarter$.getApplication(ApplicationStarter.scala:56)
	at com.jpro.internal.config.ApplicationStarter$.$anonfun$getApplication$2(ApplicationStarter.scala:54)
	at scala.Option.map(Option.scala:230)
	at com.jpro.internal.config.ApplicationStarter$.appor$lzycompute$1(ApplicationStarter.scala:54)
	at com.jpro.internal.config.ApplicationStarter$.appor$1(ApplicationStarter.scala:54)
	at com.jpro.internal.config.ApplicationStarter$.$anonfun$getApplication$3(ApplicationStarter.scala:55)
	at scala.Option.orElse(Option.scala:447)
	at com.jpro.internal.config.ApplicationStarter$.getApplication(ApplicationStarter.scala:55)
	at com.jpro.internal.communicator.ViewImpl.<init>(ViewImpl.scala:30)
	at com.jpro.internal.server.ViewByActor.<init>(ViewSerializerActor.scala:116)
	at com.jpro.internal.server.ViewSerializerActor$$anonfun$receive$2$$anonfun$applyOrElse$1.$anonfun$applyOrElse$2(ViewSerializerActor.scala:70)
	at simplefx.cores.standard.parts.CoreTimeEvent$$anon$3.run(CoreTimeEvent.scala:62)
	at com.sun.javafx.application.PlatformImpl.lambda$runLater$10(PlatformImpl.java:457)
	at java.base/java.security.AccessController.doPrivileged(Native Method)
	at com.sun.javafx.application.PlatformImpl.lambda$runLater$11(PlatformImpl.java:456)
	at com.sun.glass.ui.monocle.RunnableProcessor.runLoop(RunnableProcessor.java:92)
	at com.sun.glass.ui.monocle.RunnableProcessor.run(RunnableProcessor.java:51)
	at java.base/java.lang.Thread.run(Thread.java:834)
2022-06-25 14:11:07,755 [ERROR] from com.jpro.internal.server.ViewSerializerActor in JavaFX Application Thread - JavaFXStreamer - got Exception: 
java.lang.AssertionError: assertion failed: Couldn't find the instance for: AppByID(808093390,None)
	at scala.Predef$.assert(Predef.scala:223)
	at com.jpro.internal.config.ApplicationStarter$.getApplication(ApplicationStarter.scala:56)
	at com.jpro.internal.config.ApplicationStarter$.$anonfun$getApplication$2(ApplicationStarter.scala:54)
	at scala.Option.map(Option.scala:230)
	at com.jpro.internal.config.ApplicationStarter$.appor$lzycompute$1(ApplicationStarter.scala:54)
	at com.jpro.internal.config.ApplicationStarter$.appor$1(ApplicationStarter.scala:54)
	at com.jpro.internal.config.ApplicationStarter$.$anonfun$getApplication$3(ApplicationStarter.scala:55)
	at scala.Option.orElse(Option.scala:447)
	at com.jpro.internal.config.ApplicationStarter$.getApplication(ApplicationStarter.scala:55)
	at com.jpro.internal.communicator.ViewImpl.<init>(ViewImpl.scala:30)
	at com.jpro.internal.server.ViewByActor.<init>(ViewSerializerActor.scala:116)
	at com.jpro.internal.server.ViewSerializerActor$$anonfun$receive$2$$anonfun$applyOrElse$1.$anonfun$applyOrElse$2(ViewSerializerActor.scala:70)
	at simplefx.cores.standard.parts.CoreTimeEvent$$anon$3.run(CoreTimeEvent.scala:62)
	at com.sun.javafx.application.PlatformImpl.lambda$runLater$10(PlatformImpl.java:457)
	at java.base/java.security.AccessController.doPrivileged(Native Method)
	at com.sun.javafx.application.PlatformImpl.lambda$runLater$11(PlatformImpl.java:456)
	at com.sun.glass.ui.monocle.RunnableProcessor.runLoop(RunnableProcessor.java:92)
	at com.sun.glass.ui.monocle.RunnableProcessor.run(RunnableProcessor.java:51)
	at java.base/java.lang.Thread.run(Thread.java:834)
2022-06-25 14:11:09,413 [WARN] from org.hibernate.orm.connections.pooling in JavaFX Application Thread - HHH10001002: Using Hibernate built-in connection pool (not for production use!)
2022-06-25 14:11:09,589 [WARN] from com.jpro.internal.communicator.InstanceImpl in JavaFX Application Thread - Application didn't show the stage. JPro is now calling show()
2022-06-25 14:11:11,563 [WARN] from com.jpro.internal.server.ViewSerializerActor in application-akka.actor.default-dispatcher-10 - Unexpected message form client, closing session.
2022-06-25 14:11:12,086 [WARN] from com.jpro.internal.server.ViewSerializerActor in application-akka.actor.default-dispatcher-12 - Unexpected message form client, closing session.
2022-06-25 14:39:06,043 [WARN] from javafx in application-akka.actor.default-dispatcher-8 - Unsupported JavaFX configuration: classes were loaded from 'unnamed module @1b74cf9f'
2022-06-25 14:39:10,997 [WARN] from org.hibernate.orm.connections.pooling in JavaFX Application Thread - HHH10001002: Using Hibernate built-in connection pool (not for production use!)
2022-06-25 14:39:13,569 [WARN] from com.jpro.internal.communicator.InstanceImpl in JavaFX Application Thread - Application didn't show the stage. JPro is now calling show()
2022-06-25 14:39:23,446 [WARN] from org.hibernate.orm.connections.pooling in JavaFX Application Thread - HHH10001002: Using Hibernate built-in connection pool (not for production use!)
2022-06-25 14:39:33,099 [WARN] from org.hibernate.orm.connections.pooling in JavaFX Application Thread - HHH10001002: Using Hibernate built-in connection pool (not for production use!)
2022-06-25 14:39:34,622 [WARN] from com.jpro.internal.server.ViewSerializerActor in application-akka.actor.default-dispatcher-8 - Unexpected message form client, closing session.
2022-06-25 14:39:36,113 [WARN] from com.jpro.internal.server.ViewSerializerActor in application-akka.actor.default-dispatcher-11 - Unexpected message form client, closing session.
2022-06-25 14:39:45,392 [WARN] from org.hibernate.orm.connections.pooling in JavaFX Application Thread - HHH10001002: Using Hibernate built-in connection pool (not for production use!)
2022-06-25 14:39:45,575 [WARN] from com.jpro.meta.stream.FXAppSerializer in JavaFX Application Thread - On setting focus to window: peer was null
2022-06-25 14:39:45,789 [WARN] from com.jpro.meta.stream.FXAppSerializer in JavaFX Application Thread - On setting focus to window: peer was null
2022-06-25 14:39:45,996 [WARN] from com.jpro.meta.stream.FXAppSerializer in JavaFX Application Thread - On setting focus to window: peer was null
2022-06-25 14:39:46,204 [WARN] from com.jpro.meta.stream.FXAppSerializer in JavaFX Application Thread - On setting focus to window: peer was null
2022-06-25 14:39:46,413 [WARN] from com.jpro.meta.stream.FXAppSerializer in JavaFX Application Thread - On setting focus to window: peer was null
2022-06-25 14:39:46,619 [WARN] from com.jpro.meta.stream.FXAppSerializer in JavaFX Application Thread - On setting focus to window: peer was null
2022-06-25 14:39:46,829 [WARN] from com.jpro.meta.stream.FXAppSerializer in JavaFX Application Thread - On setting focus to window: peer was null
2022-06-25 14:39:47,036 [WARN] from com.jpro.meta.stream.FXAppSerializer in JavaFX Application Thread - On setting focus to window: peer was null
2022-06-25 14:39:47,244 [WARN] from com.jpro.meta.stream.FXAppSerializer in JavaFX Application Thread - On setting focus to window: peer was null
2022-06-25 14:39:47,452 [WARN] from com.jpro.meta.stream.FXAppSerializer in JavaFX Application Thread - On setting focus to window: peer was null
2022-06-25 14:39:47,660 [WARN] from com.jpro.meta.stream.FXAppSerializer in JavaFX Application Thread - On setting focus to window: peer was null
2022-06-25 14:39:47,867 [WARN] from com.jpro.meta.stream.FXAppSerializer in JavaFX Application Thread - On setting focus to window: peer was null
2022-06-25 14:39:48,076 [WARN] from com.jpro.meta.stream.FXAppSerializer in JavaFX Application Thread - On setting focus to window: peer was null
2022-06-25 14:39:48,285 [WARN] from com.jpro.meta.stream.FXAppSerializer in JavaFX Application Thread - On setting focus to window: peer was null
2022-06-25 14:39:48,377 [WARN] from com.jpro.internal.server.ViewSerializerActor in application-akka.actor.default-dispatcher-8 - Unexpected message form client, closing session.
2022-06-25 14:39:51,009 [WARN] from com.jpro.internal.server.ViewSerializerActor in application-akka.actor.default-dispatcher-8 - Unexpected message form client, closing session.
2022-06-25 14:39:53,065 [WARN] from com.jpro.internal.server.ViewSerializerActor in application-akka.actor.default-dispatcher-13 - Unexpected message form client, closing session.
2022-06-25 14:39:57,162 [WARN] from com.jpro.internal.server.ViewSerializerActor in application-akka.actor.default-dispatcher-7 - Unexpected message form client, closing session.
2022-06-25 14:45:43,103 [WARN] from javafx in application-akka.actor.default-dispatcher-7 - Unsupported JavaFX configuration: classes were loaded from 'unnamed module @504718d'
2022-06-25 14:45:48,440 [WARN] from org.hibernate.orm.connections.pooling in JavaFX Application Thread - HHH10001002: Using Hibernate built-in connection pool (not for production use!)
2022-06-25 14:45:50,951 [WARN] from com.jpro.internal.communicator.InstanceImpl in JavaFX Application Thread - Application didn't show the stage. JPro is now calling show()
2022-06-25 14:45:57,953 [WARN] from com.jpro.internal.server.ViewSerializerActor in application-akka.actor.default-dispatcher-9 - Unexpected message form client, closing session.
2022-06-25 14:45:58,834 [WARN] from com.jpro.internal.server.ViewSerializerActor in application-akka.actor.default-dispatcher-9 - Unexpected message form client, closing session.
2022-06-25 14:46:00,953 [ERROR] from com.jpro.internal.server.ViewSerializerActor in JavaFX Application Thread - JavaFXStreamer - got Exception: 
java.lang.AssertionError: assertion failed: Couldn't find the instance for: AppByID(8511801,None)
	at scala.Predef$.assert(Predef.scala:223)
	at com.jpro.internal.config.ApplicationStarter$.getApplication(ApplicationStarter.scala:56)
	at com.jpro.internal.config.ApplicationStarter$.$anonfun$getApplication$2(ApplicationStarter.scala:54)
	at scala.Option.map(Option.scala:230)
	at com.jpro.internal.config.ApplicationStarter$.appor$lzycompute$1(ApplicationStarter.scala:54)
	at com.jpro.internal.config.ApplicationStarter$.appor$1(ApplicationStarter.scala:54)
	at com.jpro.internal.config.ApplicationStarter$.$anonfun$getApplication$3(ApplicationStarter.scala:55)
	at scala.Option.orElse(Option.scala:447)
	at com.jpro.internal.config.ApplicationStarter$.getApplication(ApplicationStarter.scala:55)
	at com.jpro.internal.communicator.ViewImpl.<init>(ViewImpl.scala:30)
	at com.jpro.internal.server.ViewByActor.<init>(ViewSerializerActor.scala:116)
	at com.jpro.internal.server.ViewSerializerActor$$anonfun$receive$2$$anonfun$applyOrElse$1.$anonfun$applyOrElse$2(ViewSerializerActor.scala:70)
	at simplefx.cores.standard.parts.CoreTimeEvent$$anon$3.run(CoreTimeEvent.scala:62)
	at com.sun.javafx.application.PlatformImpl.lambda$runLater$10(PlatformImpl.java:457)
	at java.base/java.security.AccessController.doPrivileged(Native Method)
	at com.sun.javafx.application.PlatformImpl.lambda$runLater$11(PlatformImpl.java:456)
	at com.sun.glass.ui.monocle.RunnableProcessor.runLoop(RunnableProcessor.java:92)
	at com.sun.glass.ui.monocle.RunnableProcessor.run(RunnableProcessor.java:51)
	at java.base/java.lang.Thread.run(Thread.java:834)
2022-06-25 14:46:00,989 [ERROR] from com.jpro.internal.server.ViewSerializerActor in JavaFX Application Thread - JavaFXStreamer - got Exception: 
java.lang.AssertionError: assertion failed: Couldn't find the instance for: AppByID(140726440,None)
	at scala.Predef$.assert(Predef.scala:223)
	at com.jpro.internal.config.ApplicationStarter$.getApplication(ApplicationStarter.scala:56)
	at com.jpro.internal.config.ApplicationStarter$.$anonfun$getApplication$2(ApplicationStarter.scala:54)
	at scala.Option.map(Option.scala:230)
	at com.jpro.internal.config.ApplicationStarter$.appor$lzycompute$1(ApplicationStarter.scala:54)
	at com.jpro.internal.config.ApplicationStarter$.appor$1(ApplicationStarter.scala:54)
	at com.jpro.internal.config.ApplicationStarter$.$anonfun$getApplication$3(ApplicationStarter.scala:55)
	at scala.Option.orElse(Option.scala:447)
	at com.jpro.internal.config.ApplicationStarter$.getApplication(ApplicationStarter.scala:55)
	at com.jpro.internal.communicator.ViewImpl.<init>(ViewImpl.scala:30)
	at com.jpro.internal.server.ViewByActor.<init>(ViewSerializerActor.scala:116)
	at com.jpro.internal.server.ViewSerializerActor$$anonfun$receive$2$$anonfun$applyOrElse$1.$anonfun$applyOrElse$2(ViewSerializerActor.scala:70)
	at simplefx.cores.standard.parts.CoreTimeEvent$$anon$3.run(CoreTimeEvent.scala:62)
	at com.sun.javafx.application.PlatformImpl.lambda$runLater$10(PlatformImpl.java:457)
	at java.base/java.security.AccessController.doPrivileged(Native Method)
	at com.sun.javafx.application.PlatformImpl.lambda$runLater$11(PlatformImpl.java:456)
	at com.sun.glass.ui.monocle.RunnableProcessor.runLoop(RunnableProcessor.java:92)
	at com.sun.glass.ui.monocle.RunnableProcessor.run(RunnableProcessor.java:51)
	at java.base/java.lang.Thread.run(Thread.java:834)
2022-06-25 14:46:01,019 [ERROR] from com.jpro.internal.server.ViewSerializerActor in JavaFX Application Thread - JavaFXStreamer - got Exception: 
java.lang.AssertionError: assertion failed: Couldn't find the instance for: AppByID(808093390,None)
	at scala.Predef$.assert(Predef.scala:223)
	at com.jpro.internal.config.ApplicationStarter$.getApplication(ApplicationStarter.scala:56)
	at com.jpro.internal.config.ApplicationStarter$.$anonfun$getApplication$2(ApplicationStarter.scala:54)
	at scala.Option.map(Option.scala:230)
	at com.jpro.internal.config.ApplicationStarter$.appor$lzycompute$1(ApplicationStarter.scala:54)
	at com.jpro.internal.config.ApplicationStarter$.appor$1(ApplicationStarter.scala:54)
	at com.jpro.internal.config.ApplicationStarter$.$anonfun$getApplication$3(ApplicationStarter.scala:55)
	at scala.Option.orElse(Option.scala:447)
	at com.jpro.internal.config.ApplicationStarter$.getApplication(ApplicationStarter.scala:55)
	at com.jpro.internal.communicator.ViewImpl.<init>(ViewImpl.scala:30)
	at com.jpro.internal.server.ViewByActor.<init>(ViewSerializerActor.scala:116)
	at com.jpro.internal.server.ViewSerializerActor$$anonfun$receive$2$$anonfun$applyOrElse$1.$anonfun$applyOrElse$2(ViewSerializerActor.scala:70)
	at simplefx.cores.standard.parts.CoreTimeEvent$$anon$3.run(CoreTimeEvent.scala:62)
	at com.sun.javafx.application.PlatformImpl.lambda$runLater$10(PlatformImpl.java:457)
	at java.base/java.security.AccessController.doPrivileged(Native Method)
	at com.sun.javafx.application.PlatformImpl.lambda$runLater$11(PlatformImpl.java:456)
	at com.sun.glass.ui.monocle.RunnableProcessor.runLoop(RunnableProcessor.java:92)
	at com.sun.glass.ui.monocle.RunnableProcessor.run(RunnableProcessor.java:51)
	at java.base/java.lang.Thread.run(Thread.java:834)
2022-06-25 14:46:01,053 [ERROR] from com.jpro.internal.server.ViewSerializerActor in JavaFX Application Thread - JavaFXStreamer - got Exception: 
java.lang.AssertionError: assertion failed: Couldn't find the instance for: AppByID(1452336998,None)
	at scala.Predef$.assert(Predef.scala:223)
	at com.jpro.internal.config.ApplicationStarter$.getApplication(ApplicationStarter.scala:56)
	at com.jpro.internal.config.ApplicationStarter$.$anonfun$getApplication$2(ApplicationStarter.scala:54)
	at scala.Option.map(Option.scala:230)
	at com.jpro.internal.config.ApplicationStarter$.appor$lzycompute$1(ApplicationStarter.scala:54)
	at com.jpro.internal.config.ApplicationStarter$.appor$1(ApplicationStarter.scala:54)
	at com.jpro.internal.config.ApplicationStarter$.$anonfun$getApplication$3(ApplicationStarter.scala:55)
	at scala.Option.orElse(Option.scala:447)
	at com.jpro.internal.config.ApplicationStarter$.getApplication(ApplicationStarter.scala:55)
	at com.jpro.internal.communicator.ViewImpl.<init>(ViewImpl.scala:30)
	at com.jpro.internal.server.ViewByActor.<init>(ViewSerializerActor.scala:116)
	at com.jpro.internal.server.ViewSerializerActor$$anonfun$receive$2$$anonfun$applyOrElse$1.$anonfun$applyOrElse$2(ViewSerializerActor.scala:70)
	at simplefx.cores.standard.parts.CoreTimeEvent$$anon$3.run(CoreTimeEvent.scala:62)
	at com.sun.javafx.application.PlatformImpl.lambda$runLater$10(PlatformImpl.java:457)
	at java.base/java.security.AccessController.doPrivileged(Native Method)
	at com.sun.javafx.application.PlatformImpl.lambda$runLater$11(PlatformImpl.java:456)
	at com.sun.glass.ui.monocle.RunnableProcessor.runLoop(RunnableProcessor.java:92)
	at com.sun.glass.ui.monocle.RunnableProcessor.run(RunnableProcessor.java:51)
	at java.base/java.lang.Thread.run(Thread.java:834)
2022-06-25 14:46:15,884 [ERROR] from com.jpro.internal.server.ViewSerializerActor in JavaFX Application Thread - JavaFXStreamer - got Exception: 
java.lang.AssertionError: assertion failed: Couldn't find the instance for: AppByID(8511801,None)
	at scala.Predef$.assert(Predef.scala:223)
	at com.jpro.internal.config.ApplicationStarter$.getApplication(ApplicationStarter.scala:56)
	at com.jpro.internal.config.ApplicationStarter$.$anonfun$getApplication$2(ApplicationStarter.scala:54)
	at scala.Option.map(Option.scala:230)
	at com.jpro.internal.config.ApplicationStarter$.appor$lzycompute$1(ApplicationStarter.scala:54)
	at com.jpro.internal.config.ApplicationStarter$.appor$1(ApplicationStarter.scala:54)
	at com.jpro.internal.config.ApplicationStarter$.$anonfun$getApplication$3(ApplicationStarter.scala:55)
	at scala.Option.orElse(Option.scala:447)
	at com.jpro.internal.config.ApplicationStarter$.getApplication(ApplicationStarter.scala:55)
	at com.jpro.internal.communicator.ViewImpl.<init>(ViewImpl.scala:30)
	at com.jpro.internal.server.ViewByActor.<init>(ViewSerializerActor.scala:116)
	at com.jpro.internal.server.ViewSerializerActor$$anonfun$receive$2$$anonfun$applyOrElse$1.$anonfun$applyOrElse$2(ViewSerializerActor.scala:70)
	at simplefx.cores.standard.parts.CoreTimeEvent$$anon$3.run(CoreTimeEvent.scala:62)
	at com.sun.javafx.application.PlatformImpl.lambda$runLater$10(PlatformImpl.java:457)
	at java.base/java.security.AccessController.doPrivileged(Native Method)
	at com.sun.javafx.application.PlatformImpl.lambda$runLater$11(PlatformImpl.java:456)
	at com.sun.glass.ui.monocle.RunnableProcessor.runLoop(RunnableProcessor.java:92)
	at com.sun.glass.ui.monocle.RunnableProcessor.run(RunnableProcessor.java:51)
	at java.base/java.lang.Thread.run(Thread.java:834)
2022-06-25 14:46:15,916 [ERROR] from com.jpro.internal.server.ViewSerializerActor in JavaFX Application Thread - JavaFXStreamer - got Exception: 
java.lang.AssertionError: assertion failed: Couldn't find the instance for: AppByID(140726440,None)
	at scala.Predef$.assert(Predef.scala:223)
	at com.jpro.internal.config.ApplicationStarter$.getApplication(ApplicationStarter.scala:56)
	at com.jpro.internal.config.ApplicationStarter$.$anonfun$getApplication$2(ApplicationStarter.scala:54)
	at scala.Option.map(Option.scala:230)
	at com.jpro.internal.config.ApplicationStarter$.appor$lzycompute$1(ApplicationStarter.scala:54)
	at com.jpro.internal.config.ApplicationStarter$.appor$1(ApplicationStarter.scala:54)
	at com.jpro.internal.config.ApplicationStarter$.$anonfun$getApplication$3(ApplicationStarter.scala:55)
	at scala.Option.orElse(Option.scala:447)
	at com.jpro.internal.config.ApplicationStarter$.getApplication(ApplicationStarter.scala:55)
	at com.jpro.internal.communicator.ViewImpl.<init>(ViewImpl.scala:30)
	at com.jpro.internal.server.ViewByActor.<init>(ViewSerializerActor.scala:116)
	at com.jpro.internal.server.ViewSerializerActor$$anonfun$receive$2$$anonfun$applyOrElse$1.$anonfun$applyOrElse$2(ViewSerializerActor.scala:70)
	at simplefx.cores.standard.parts.CoreTimeEvent$$anon$3.run(CoreTimeEvent.scala:62)
	at com.sun.javafx.application.PlatformImpl.lambda$runLater$10(PlatformImpl.java:457)
	at java.base/java.security.AccessController.doPrivileged(Native Method)
	at com.sun.javafx.application.PlatformImpl.lambda$runLater$11(PlatformImpl.java:456)
	at com.sun.glass.ui.monocle.RunnableProcessor.runLoop(RunnableProcessor.java:92)
	at com.sun.glass.ui.monocle.RunnableProcessor.run(RunnableProcessor.java:51)
	at java.base/java.lang.Thread.run(Thread.java:834)
2022-06-25 14:46:15,946 [ERROR] from com.jpro.internal.server.ViewSerializerActor in JavaFX Application Thread - JavaFXStreamer - got Exception: 
java.lang.AssertionError: assertion failed: Couldn't find the instance for: AppByID(1452336998,None)
	at scala.Predef$.assert(Predef.scala:223)
	at com.jpro.internal.config.ApplicationStarter$.getApplication(ApplicationStarter.scala:56)
	at com.jpro.internal.config.ApplicationStarter$.$anonfun$getApplication$2(ApplicationStarter.scala:54)
	at scala.Option.map(Option.scala:230)
	at com.jpro.internal.config.ApplicationStarter$.appor$lzycompute$1(ApplicationStarter.scala:54)
	at com.jpro.internal.config.ApplicationStarter$.appor$1(ApplicationStarter.scala:54)
	at com.jpro.internal.config.ApplicationStarter$.$anonfun$getApplication$3(ApplicationStarter.scala:55)
	at scala.Option.orElse(Option.scala:447)
	at com.jpro.internal.config.ApplicationStarter$.getApplication(ApplicationStarter.scala:55)
	at com.jpro.internal.communicator.ViewImpl.<init>(ViewImpl.scala:30)
	at com.jpro.internal.server.ViewByActor.<init>(ViewSerializerActor.scala:116)
	at com.jpro.internal.server.ViewSerializerActor$$anonfun$receive$2$$anonfun$applyOrElse$1.$anonfun$applyOrElse$2(ViewSerializerActor.scala:70)
	at simplefx.cores.standard.parts.CoreTimeEvent$$anon$3.run(CoreTimeEvent.scala:62)
	at com.sun.javafx.application.PlatformImpl.lambda$runLater$10(PlatformImpl.java:457)
	at java.base/java.security.AccessController.doPrivileged(Native Method)
	at com.sun.javafx.application.PlatformImpl.lambda$runLater$11(PlatformImpl.java:456)
	at com.sun.glass.ui.monocle.RunnableProcessor.runLoop(RunnableProcessor.java:92)
	at com.sun.glass.ui.monocle.RunnableProcessor.run(RunnableProcessor.java:51)
	at java.base/java.lang.Thread.run(Thread.java:834)
2022-06-25 14:46:15,979 [ERROR] from com.jpro.internal.server.ViewSerializerActor in JavaFX Application Thread - JavaFXStreamer - got Exception: 
java.lang.AssertionError: assertion failed: Couldn't find the instance for: AppByID(808093390,None)
	at scala.Predef$.assert(Predef.scala:223)
	at com.jpro.internal.config.ApplicationStarter$.getApplication(ApplicationStarter.scala:56)
	at com.jpro.internal.config.ApplicationStarter$.$anonfun$getApplication$2(ApplicationStarter.scala:54)
	at scala.Option.map(Option.scala:230)
	at com.jpro.internal.config.ApplicationStarter$.appor$lzycompute$1(ApplicationStarter.scala:54)
	at com.jpro.internal.config.ApplicationStarter$.appor$1(ApplicationStarter.scala:54)
	at com.jpro.internal.config.ApplicationStarter$.$anonfun$getApplication$3(ApplicationStarter.scala:55)
	at scala.Option.orElse(Option.scala:447)
	at com.jpro.internal.config.ApplicationStarter$.getApplication(ApplicationStarter.scala:55)
	at com.jpro.internal.communicator.ViewImpl.<init>(ViewImpl.scala:30)
	at com.jpro.internal.server.ViewByActor.<init>(ViewSerializerActor.scala:116)
	at com.jpro.internal.server.ViewSerializerActor$$anonfun$receive$2$$anonfun$applyOrElse$1.$anonfun$applyOrElse$2(ViewSerializerActor.scala:70)
	at simplefx.cores.standard.parts.CoreTimeEvent$$anon$3.run(CoreTimeEvent.scala:62)
	at com.sun.javafx.application.PlatformImpl.lambda$runLater$10(PlatformImpl.java:457)
	at java.base/java.security.AccessController.doPrivileged(Native Method)
	at com.sun.javafx.application.PlatformImpl.lambda$runLater$11(PlatformImpl.java:456)
	at com.sun.glass.ui.monocle.RunnableProcessor.runLoop(RunnableProcessor.java:92)
	at com.sun.glass.ui.monocle.RunnableProcessor.run(RunnableProcessor.java:51)
	at java.base/java.lang.Thread.run(Thread.java:834)
2022-06-25 14:46:46,208 [WARN] from javafx in application-akka.actor.default-dispatcher-8 - Unsupported JavaFX configuration: classes were loaded from 'unnamed module @4097ef5d'
2022-06-25 14:46:51,237 [WARN] from org.hibernate.orm.connections.pooling in JavaFX Application Thread - HHH10001002: Using Hibernate built-in connection pool (not for production use!)
2022-06-25 14:46:53,607 [WARN] from com.jpro.internal.communicator.InstanceImpl in JavaFX Application Thread - Application didn't show the stage. JPro is now calling show()
2022-06-25 14:46:59,938 [WARN] from com.jpro.internal.server.ViewSerializerActor in application-akka.actor.default-dispatcher-12 - Unexpected message form client, closing session.
2022-06-25 14:47:00,449 [WARN] from com.jpro.internal.server.ViewSerializerActor in application-akka.actor.default-dispatcher-10 - Unexpected message form client, closing session.
2022-06-25 14:47:03,880 [WARN] from com.jpro.internal.server.ViewSerializerActor in application-akka.actor.default-dispatcher-12 - Unexpected message form client, closing session.
